0033-relationship-between-protocols/0033-relationship-between-protocols.md

```diff
@@ -0,0 +1,115 @@
+
+## Scope
+The scope of this document is the explication of the relationship between the Interledger protocols, and the details for it are out of scope. Some references are given for it though. i.e. This document draws a whole, big picture of ILP.
+
+## Perspective
+There are two diagrams, one is for the macro view that shows the main elements and its IP layer connections, another is for the micro view that shows protocols, the interaction between programs, and the other details.
+
+The alphabets (A-C) and numbers (1-15), that are shown in the diagrams as **red characters**, are explained briefly in the [Elements and Brief Explanations](#elements-and-brief-explanations) section.
+
+### Connections
+
+![Perspective of connections](images/perspective-connections.svg)
+
+### Protocols and Details
+
+This is an enlarged diagram of the above diagram, that shows mainly `Node A`, `Node B`, and `Ledger B`.
+
+![Perspective of connections](images/perspective-protocols.svg)
+
+### Elements and Brief Explanations
+#### Connections
+- (A) BTP over WebSocket
+  - **Node** to **Node** connection
+  - In order to exchange payments, configuration and routing information, a node MUST have secure communication channels with its peers. The current implementation uses WebSockets for it.
+- (B) Ledger specific connection
+  - **Node** to **Ledger** connection
+  - A node needs the means to settle its payments. Therefore the node has a connection to a ledger. The type of connection varies depending on the ledger.
+  - In the cases where blockchains are used for settlements, nodes MAY utilize payment channels because it is quick and inexpensive than issuing transactions every time the nodes want to settle.
+- (C) SPSP over HTTPS
+  - **Application** to **SPSP Server** connection
+  - To determine end-to-end payment information such as a shared secret, a destination address and so forth, an application connects to SPSP server over HTTPS.
+
+#### Protocols
+- Link Layer Protocol
+  - (1) BTP
+    - [Bilateral Transfer Protocol 2.0 (BTP/2.0)](../0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md)
+    - BTP is used for transferring ILP packets and messages that are used for settlements and so on between two nodes.
+- Core ILP Protocols
+  - (2) ILP
+    - [Interledger Protocol V4 (ILPv4)](../0027-interledger-protocol-4/0027-interledger-protocol-4.md)
+    - ILP is used for sending payment packets across multiple hops. Some other protocols, including those for node configuration and routing, are also built on top of ILP and use ILP packets to communicate that information between peers.
+  - (10) ILP Address
```

Comment:
Seems a bit odd that these are numbered out of order

Reply:
I fixed as far as I can (packet structure may look a bit odd because it lacks `3`)